pennylane/operation.py

```diff
@@ -627,10 +627,12 @@ def inv(self):
 
     @property
     def matrix(self):
+        op_matrix = self._matrix(*self.parameters)
+
         if self.inverse:
-            return np.linalg.inv(self._matrix(*self.parameters))
+            return op_matrix.conj().T
 
-        return self._matrix(*self.parameters)
+        return op_matrix
```

Comment:
Cool! :)

Reply:
Yes, that should also make inverses on default.qubit notably faster!